tools/meson.build

```diff
@@ -0,0 +1,29 @@
+#  Copyright © 2022 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+inc = include_directories('../include')
+
+lib_m = cc.find_library('m')
+
+executable('elc3', ['elc3.c', 'lc3bin.c', 'wave.c'],
+    link_with : lc3lib,
+    link_args : ['-flto'],
+    include_directories: inc,
+    dependencies: lib_m)
+
+executable('dlc3', ['dlc3.c', 'lc3bin.c', 'wave.c'],
+    link_with : lc3lib,
+    link_args : ['-flto'],
```

Comment:
Meson has a builtin `-Db_lto=false` option, so you want to just set that in the main `project(..., default_options: ['b_lto=true'])` and allow Meson to portably do this for you.

Note also that Meson does it *correctly*, i.e. by setting the right compile args too. :)